src/commands/remoteDebug/getRemoteDebugLanguage.ts

```diff
@@ -50,8 +50,8 @@ export function getRemoteDebugLanguage(siteConfig: SiteConfigResource, context:
 
 // Remote debugging is currently only supported for Node.js >= 8.11
```

Comment:
I assume the portal doesn't support 6-lts? And that 8-lts actually resolves to a version greater than 8.11?